# gets the current pressure in Millibars from the pressure sensor
---
float64 pressure